whylog/teacher/__init__.py

```diff
@@ -52,4 +56,4 @@ def set_causes_relation(self, relation):
         pass
 
     def save(self):
-        pass
+        self.config.add_rule(create_sample_rule())
```

Comment:
This line suggests that the caller of config does not need to call `config.save()` after finishing modifying configuration, which will require O(N) rewrites of the config to storage (file/database/etc). Maybe it should be a context manager (which translates to open file for writing / transaction) or something?